com.visualstudio.code.tool.git-lfs.yml

```diff
@@ -0,0 +1,51 @@
+id: com.visualstudio.code.tool.git-lfs
+branch: '22.08'
+build-extension: true
+sdk: org.freedesktop.Sdk//22.08
```

Comment:
```suggestion
branch: '23.08'
build-extension: true
sdk: org.freedesktop.Sdk//23.08
```

Reply:
NO.

The host app is still 22.08

Reply:
Hmm, thought they updated to 22.08 by now. Ignore.